src/recipe/parser/system_info.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+
+/// The system information for the rendered recipe.
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct SystemInfo {
+    /// `rattler-build` version.
+    rattler_build_version: String,
+    /// Git version.
+    git_version: Option<String>,
+    /// `patchelf` version (Linux).
+    patchelf_version: Option<String>,
+    /// `install_name_tool` version (macOS).
+    install_name_tool_version: Option<String>,
+}
+
+impl SystemInfo {
+    /// Collects the system information.
+    #[cfg(not(test))]
+    pub fn new() -> Self {
+        let mut system_info = SystemInfo {
+            rattler_build_version: env!("CARGO_PKG_VERSION").to_string(),
+            ..Default::default()
+        };
+        system_info.git_version = Self::run_os_command("git", "version");
+        if cfg!(target_os = "linux") {
```

Comment:
This is not correct, one could have `patchelf` on macOS as well: https://prefix.dev/channels/conda-forge/packages/patchelf

We could check the `target_platform` of the `output` to figure out wether we are compiling for macOS or Linux and based on that information only log the relevant one.

Reply:
Ah, good point!

> We could check the target_platform of the output to figure out wether we are compiling for macOS or Linux and based on that information only log the relevant one.

Any idea how to access that information inside `Recipe`? 👀

Reply:
Now that we will remove `install_name_tool`, does that mean we can store patchelf version for both platforms? What do you prefer here?